commands/verify/verify.go

```diff
@@ -101,8 +101,13 @@ func DoVerifyAccordingConfig() error {
 		return err
 	}
 
+	failFast := e2eConfig.Verify.Setting.FailFast
+
 	for idx, v := range e2eConfig.Verify.Cases {
 		if v.GetExpected() == "" {
+			if !failFast {
+				continue
+			}
 			return fmt.Errorf("the expected data file for case[%v] is not specified", idx)
```

Comment:
We should let users know that the current case fails even `failFast` is `False`, like this:
```go
err := fmt.Errorf("the expected data file for case[%v] is not specified", idx)
if !failFast {
    logger.Log.Warn(...)
    continue
}
return err
```

Reply:
Thanks for your advice !